float64 yaw_rate
float64 vehicleSpeed
